docs-chef-io/content/inspec/dsl_inspec.md

```diff
@@ -249,6 +249,21 @@ control 'nutcracker-connect-redis-001' do
 end
 ```
 
+Here is another simple example, which checks for, if certain pip packages are installed, only if /root/.aws exists
```

Comment:
This is a valid example of `only_if` - I wanted to confirm, is there something about this case that technically differs from the previous example? It looks like (and your description for the example says) this is simply one more example, but I wanted to make sure. If there's something that doesn't work in the previous example that this example solves, we should highlight that.

Reply:
@james-stocks this additional example, I felt, explains better (better because, at first glance, the example was not explanatory enough to understand how it worked), and also provides an example to check directory exists as a condition.